src/lib/core/ripple/ripple-ref.ts

```diff
@@ -0,0 +1,26 @@
+import {RippleConfig, RippleRenderer} from './ripple-renderer';
+
+/**
+ * Reference to a previously launched ripple element.
+ */
+export class RippleRef {
+
+  constructor(
+    private renderer: RippleRenderer,
+    public element: HTMLElement,
+    public config: RippleConfig) {
+  }
+
+  /** Fades out the ripple element. */
+  fadeOut() {
+    let rippleIndex = this.renderer.activeRipples.indexOf(this);
+
+    // Remove the ripple reference if added to the list of active ripples.
+    if (rippleIndex !== -1) {
+      this.renderer.activeRipples.splice(rippleIndex, 1);
```

Comment:
It might be better if we did this inside the `renderer.fadeOutRipple` since the renderer is the one responsible for keeping track of the ripples.